library/dnet.c

```diff
@@ -314,6 +314,31 @@ int dnet_send_reply(void *state, struct dnet_cmd *cmd, const void *odata, unsign
 	return err;
 }
 
+void dnet_queue_wait_threshold(void *state)
+{
+	struct dnet_net_state *st = state;
```

Comment:
Is it really needed to call `dnet_queue_wait_threshold` by backend? It is called only by `dnet_send_reply_threshold`, so it can be defined as a static.